wikitools/api.py

```diff
@@ -317,7 +317,8 @@ def __exceptionFromAPIError(self, data, content):
 			raise APIError(data)
 		error = content['error']['code']
 		if error == "maxlag":
-			lagtime = int(re.search("(\d+) seconds", content['error']['info']).group(1))
+			lagtime = 0.5 + float(re.search("([0-9.-]+) seconds",
```

Comment:
Should this addition of "0.5" have an inline code comment? It seems non-obvious to me.